packages/components/src/theme/ComposerTheme/Components/Input.ts

```diff
@@ -22,6 +22,7 @@ const iconContainerStyles = {
 export const iconStyles = {
   disabled: {
     color: 'text-disabled',
+    pointerEvents: 'none',
```

Comment:
Could you check if the pointerEvents is needed for the Input? I think we might only need it for the `Tag`